ui/src/pages/tasks.tsx

```
@@ -0,0 +1,48 @@
+    getDagTaskPageCount()
+      .then((count) => {
+        setMaxPage(count);
+      })
+      .catch((error) => console.error(error));
+
+  return (
+    <div class="p-4">
+      <h2 class="text-2xl font-semibold mb-4">Your Tasks</h2>
+      <div class="mt-4"></div>
+      <Show when={tasks.isError}>
+        <div>Error: {tasks.error && tasks.error.message}</div>
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)